components/storage/blockdevice/COMPONENT_SPIF/SPIFBlockDevice.cpp

```diff
@@ -211,6 +214,13 @@ int SPIFBlockDevice::init()
     _dummy_and_mode_cycles = _write_dummy_and_mode_cycles;
     _is_initialized = true;
 
+    if (_device_size_bytes > (1 << 24)) // Size is bigger than 16MB and thus address does not fit in 3 byte, switch to 4 byte address mode
```

Comment:
Nitpicking - there is no need to for the comment as you have the trace saying the same thing two lines below